drivers/sensor/qdec_s32/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config QDEC_S32
```

Comment:
Not much important but I would suggest to use `NXP_S32` instead of `S32` in the name of at least Kconfig and files, just to avoid ambiguity and consistent with the rest of NXP S32 drivers

Reply:
Changed !